docs/openapi/api-external-b2b-pa-bundle.yaml

```diff
@@ -1230,7 +1230,8 @@ components:
         subject:
           type: string
           description: titolo della notifica
-          maxLength: 134
+          minLength: 10
+          maxLength: 120
```

Comment:
max length non deve essere cambiata, bisogna controllare che venga fatto il trunk in fase di invio.

Reply:
ripristinata maxLength a 134